scripts/scripts.js

```diff
@@ -311,12 +496,14 @@ function buildArticleHeader(mainEl) {
   const div = document.createElement('div');
   const h1 = mainEl.querySelector('h1');
   const picture = mainEl.querySelector('picture');
-  const category = getMetadata('category');
+  const category = document.head.querySelector('meta[property="article:tag"]').content;
```

Comment:
could the `getMetadata` function be used here for consistency? If the passed name `article:tag` has a colon in it, `getMetadata` should return the matching meta property, right?